.github/workflows/ci.yml

```diff
@@ -0,0 +1,93 @@
+name: Alfresco Java SDK CI
+
+on:
+  push:
+    branches:
+      - release/5.1.N
+  pull_request:
+    branches:
+      - release/5.1.N
+  workflow_dispatch:
+
+env:
+  MAVEN_PASSWORD: ${{ secrets.NEXUS_PASSWORD }}
+  MAVEN_USERNAME: ${{ secrets.NEXUS_USERNAME }}
+  GITHUB_ACTIONS_DEPLOY_TIMEOUT: 60
+  CONTENT_SERVICE_SECURITY_BASICAUTH_PASSWORD: ${{ secrets.CONTENT_SERVICE_SECURITY_BASICAUTH_PASSWORD }}
+  CONTENT_SERVICE_SECURITY_BASICAUTH_USERNAME: ${{ secrets.CONTENT_SERVICE_SECURITY_BASICAUTH_USERNAME }}
+  CONTENT_SERVICE_URL: ${{ secrets.CONTENT_SERVICE_URL }}
+  ENVIRONMENT_APA_HOST: apadev-apa.envalfresco.com
+  ENVIRONMENT_HOST: apadev.envalfresco.com
+  ENVIRONMENT_APP: simpleapp
+  AGS_VERSION: 14.9
+  ACS_VERSION: 7.1.0.1
+  JAVA_VERSION: 11
+
+jobs:
+  pre-commit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: Alfresco/alfresco-build-tools/.github/actions/pre-commit@v1.35.0
```

Comment:
Let's update this tag to the latest `v1.36.0`, along with all the others in this workflow.

Reply:
done